### DISCUSSION TITLE
How to display the interface with the theme switch button on the github homepage again.

### DISCUSSION THREAD

**Discussion by aner1001** [community/community#2923] [Discussions] [answered]
Hello github,
      On the website interface, I accidentally turned off the github dark mode switch interface. How to bring up the switch interface again.

![11](https://cdn.jsdelivr.net/gh/aner1001/picBed/blog/2020/12/10/20201210154340.png)
![123](https://cdn.jsdelivr.net/gh/aner1001/picBed/blog/2020/12/10/20201210153821.png)

**Comment by becca** [comment, marked as answer]
It's under https://github.com/settings/appearance. More info can be found here https://github.blog/changelog/2020-12-08-dark-mode-public-beta/

This is a space currently exclusively dedicated to Discussions and GitHub for Mobile. Dark mode requests and bug reports go to https://github.community/c/github-help/dark-mode-beta/65

**Reply by aner1001** [reply]
I am very sorry that the question was sent in the wrong place. Thank you very much for your answers. Since I just started learning to use github, I am not familiar with the process. I will pay attention later.
